Makefile

```diff
@@ -2,7 +2,7 @@
 # Variables used for various build targets.
 ##############################################################################
 CI_ELASTIC_AGENT_DOCKER_TAG?=8.6.0-SNAPSHOT
-CI_ELASTIC_AGENT_DOCKER_IMAGE?=704479110758.dkr.ecr.eu-west-1.amazonaws.com/elastic-agent
+CI_ELASTIC_AGENT_DOCKER_IMAGE?=704479110758.dkr.ecr.eu-west-2.amazonaws.com/elastic-agent
```

Comment:
Instead of handling CI variables through this makefile, I recommend moving them into gh environments / workflows.

It feels too scattered to place these variables here